src/app/static/src/app/components/Stepper.vue

```diff
@@ -37,7 +37,7 @@
                 <download-results v-if="isActive(7)"></download-results>
             </div>
         </div>
-        <template v-if="activeStep === 3">
+        <template v-if="activeStep !== 4">
```

Comment:
This isn't quite right, because it's bundling in that warning alert which does need to go with step3 only (model Options - where we want to show warnings at the bottom). 
So I think we need split this template in two, and include only the warnings component when activeStep === 3
and, under that, another template to include stepperNavigation (and the hr above it) if activeStep !== 4